commands/box/transaction/root.go

```diff
@@ -242,6 +324,80 @@ func decoderawtx(cmd *cobra.Command, args []string) {
 	fmt.Println(util.PrettyPrint(tx))
 }
 
+func signrawtx(cmd *cobra.Command, args []string) {
+	fmt.Println("signrawtx called")
+	if len(args) != 2 {
+		fmt.Println("Invalide argument number")
+		return
+	}
+	txByte, err := hex.DecodeString(args[0])
+	if err != nil {
+		fmt.Println(err)
+		return
+	}
+	tx := new(types.Transaction)
+	if err := tx.Unmarshal(txByte); err != nil {
+		fmt.Println("Unmarshal error: ", err)
+		return
+	}
+	wltMgr, err := wallet.NewWalletManager(walletDir)
+	if err != nil {
+		fmt.Println("get wallet manager error", err)
+	}
+	accI, ok := wltMgr.GetAccount(args[1])
```

Comment:
args[1] is private key?
it is not secure!

Reply:
args[1] is address,not private key.